### DISCUSSION TITLE
Group changelog in `nx release changelog`

### DISCUSSION THREAD

**Discussion by mpsanchis** [nrwl/nx#26269] [Feature Requests]
Hi team,

Have you considered generating a changelog per-group, apart from the current cases: (1) per-project and (2) per-repo? 

I have a big repo with some groups, and each group is released together. This makes all packages from that group have the same version (like you guys 😄 ), which is very nice because imports in the `package.json` have all the same version. When I release, I'd like to have my release notes refer to the whole group, since:
* (1) having a changelog per project is an overkill, because a user of the libs would have to scroll through 10-20 changelogs
* (2) having a repo changelog doesn't make sense, because groups are not very related, and I might release only one group

I have some ideas of how to do it. If I have time, I might even do a simple custom "changelog merger" for each group, that puts all that data in a single file. However, it'd be cool to have this native.

---- 

Note: Adding the discussion again because the [original](https://github.com/nrwl/nx/discussions/23819) has disappeared. Thank you @jaysoo for replying 🙏

**Comment by JamesHenry** [comment]
I think it's a good suggestion, it makes a lot of sense for your use-case. We won't have bandwidth to work on it in the next couple of cycles, so if you want to work on something in the meantime (or even a PR to Nx) that would be great
